src/fetchUpdates.js

```diff
@@ -0,0 +1,20 @@
+const fetch = require('node-fetch');
+
+(async () => {
+  try {
+    const headers = {
+      headers: { Accept: 'application/vnd.github.v3.raw' },
+    };
+
+    const response = await fetch(
+      `https://api.github.com/repos/ApollosProject/apollos-prototype/contents/version.json`,
+      headers,
+    );
+
+    const { mostRecentVersion, newestVersion } = await response.json();
+    
+    return { mostRecentVersion, newestVersion };
+  } catch (e) {
+    throw new Error(e)
+  }
+})();
```

Comment:
I don't believe this file exports anything? How do the values in this file get exported? I might be missing something obvious.

Reply:
Nope. I completely forgot to export the IIFE.